tests/bootstrap.php

```diff
@@ -3,6 +3,8 @@
 // ensure we get report on all possible php errors
 error_reporting(E_ALL);
 
+$_SERVER['REQUEST_URI'] = '/';
```

Comment:
Why do we need it?

Reply:
You no longer need, I already delete it.